### PR TITLE
updated the meesages.cy file with welsh translation

### PR DESCRIPTION
**Pull request by DurgaJonnalagadda**

### CODE REVIEW

**Comment by sraspin** [comment]
Hi Durga, could you ensure that the Internal Server Error page also gets translated please. Tom requested that I do the same thing for Vat-correspondence. See my PR for vat-correspondence or Tom's PR in VACLUF for how to do it

File: app/config/ServiceErrorHandler.scala [hunk 1]
```diff
@@ -17,19 +17,39 @@
...
 
 import javax.inject.Inject
-
 import play.api.i18n.MessagesApi
 import play.api.mvc.Results.InternalServerError
-import play.api.mvc.{Request, Result}
+import play.api.mvc.{Request, RequestHeader, Result}
 import play.twirl.api.Html
 import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
+import views.html.errors.standardError_Scope0.standardError
```
Comment (sraspin) [inline app/config/ServiceErrorHandler.scala]: Could you change this import to: "import views.html.errors.standardError"
I'm not entirely sure of the implications of using standardError_Scope

File: app/config/ServiceErrorHandler.scala [hunk 2]
```diff
@@ -17,19 +17,39 @@
...
+
+import scala.concurrent.Future
+
+class ServiceErrorHandler @Inject()(val messagesApi: MessagesApi,
+                                    standardErrorView: standardError,
```
Comment (sraspin) [inline app/config/ServiceErrorHandler.scala]: This can be removed

File: app/config/ServiceErrorHandler.scala [hunk 3]
```diff
@@ -17,19 +17,39 @@
...
+                                    implicit val appConfig: AppConfig) extends FrontendErrorHandler {
+
+  private implicit def rhToRequest(rh: RequestHeader): Request[_] = Request(rh, "")
 
-class ServiceErrorHandler @Inject()(val messagesApi: MessagesApi, appConfig: AppConfig)
-  extends FrontendErrorHandler {
+  override def standardErrorTemplate(pageTitle: String, heading: String, message: String)
+                                    (implicit request: Request[_]): Html =
+    standardErrorView(appConfig, pageTitle, heading, message)
```
Comment (sraspin) [inline app/config/ServiceErrorHandler.scala]: Could you change this line to: standardError(appConfig, pageTitle, heading, message)

File: app/config/ServiceErrorHandler.scala (hunk 1 above)
Comment (DurgaJonnalagadda) [reply app/config/ServiceErrorHandler.scala]: updated

File: app/config/ServiceErrorHandler.scala (hunk 2 above)
Comment (DurgaJonnalagadda) [reply app/config/ServiceErrorHandler.scala]: removed

File: app/config/ServiceErrorHandler.scala (hunk 3 above)
Comment (DurgaJonnalagadda) [reply app/config/ServiceErrorHandler.scala]: updated

**Pull request merged**